endpoints/openrtb2/auction_test.go

```diff
@@ -3209,6 +3273,23 @@ var testFinalRequests = []string{
 }`,
 }
 
+var testStoredImpIds = []string{
+	"adUnit1", "adUnit2", "adUnit1", "some-static-imp",
+}
+
+var testStoredImps = []string{
+	`{
+        				"w":200,
+        				"h":300
+        			}`,
+	`{
+					"w":200,
+					"h":300
+				}`,
+	``,
+	``,
```

Comment:
Nitpick: This is harder to read with the whitespace. Please consider using just a few spaces instead?

Reply:
Formatted